libs/motor/inc/public/tfc/motor/details/positioner_impl.hpp

```diff
@@ -168,8 +159,14 @@ struct tachometer {
     statistics_.update(now);
     auto constexpr increment{ 1 };
     position_ += increment;
-    std::invoke(position_update_callback_, increment, statistics().average(), statistics().stddev(),
-                detect_deviation_from_average(statistics_.last_interval(), statistics_.average()));
+    auto err{ errors::err_enum::success };
+    // check if interval is greater than 180% of average and less than 400% of average
+    // todo is 400% good strategy to detect when stopped and started again?
+    if (statistics_.last_interval() * 10 > statistics_.average() * 18 &&
+        statistics_.last_interval() < statistics_.average() * 4) {
```

Comment:
This is confusing, where is the original 400% number coming from, what are we trying to avoid with this change. Could we express this in more clear terms.

Reply:
400% was decided based on experience, normally you could miss 2 - 3 pulses or up to 300 % +

but this strategy can be discussed, this at least is more stable than it was before. It is really weird to report missing event if motor is starting.